src/main/java/hbz/limetrans/Main.java

```diff
@@ -5,9 +5,10 @@
 import org.apache.commons.validator.routines.UrlValidator;
 import org.xbib.common.settings.Settings;
 
-import java.io.IOException;
 import java.io.File;
+import java.io.IOException;
 import java.net.URL;
+import java.util.Arrays;
 
 public final class Main {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)